tests/util/index.js

```diff
@@ -26,7 +26,7 @@ function callModule(fn, filename) {
   return module.exports;
 }
 
-exports.compile = function(fixturePath) {
+exports.compile = function(fixturePath, exportStats) {
```

Comment:
Turn `exportStats` into an options object and use a `exportStats` field on it for this. I don't want a boolean trap in this api.